### ISSUE TITLE
acq_time format

### ISSUE THREAD

**Issue by jasmainak** [mne-tools/mne-bids#71]
On `master` of bids-validator, we get the following error:

```sh
1: Entries in the "acq_time" column of _scans.tsv should be 
expressed in the following format YYYY-MM-DDTHH:mm:ss 
(year, month, day, hour (24h), minute, second; this is equivalent to the 
RFC3339 “date-time” format.  (code: 84 - ACQTIME_FMT)
```

**Comment by sappelhoff** [comment]
The errors originate from the fact, that we don't really have an `acq_time` and just write `n/a` in `test_fif` and `test_ctf`. While the BIDS-validator probably should accept an `n/a` within the `acq_time` column, we can solve this in another way:

Quoting from the [_scans section in BIDS specification](https://docs.google.com/document/d/1HFUkAEE-pB-angVcYe6pf_-fVf4sCpOHKesUvfb8Grc/edit#heading=h.rw11qtsldsuw):

> The purpose of this file is to describe timing and other properties of each imaging acquisition sequence (each run .nii[.gz] file) within one session. Each .nii[.gz] file should be described by at most one row. Relative paths to files should be used under a compulsory “filename” header. 
**If acquisition time is included it should be under “acq_time” header.** Datetime should be expressed in the following format 2009-06-15T13:45:30 (year, month, day, hour (24h), minute, second; this is equivalent to the RFC3339 “date-time” format, time zone is always assumed as local time). For anonymization purposes all dates within one subject should be shifted by a randomly chosen (but common across all runs etc.) number of days. This way relative timing would be preserved, but chances of identifying a person based on the date and time of their scan would be decreased. Dates that are shifted for anonymization purposes should be set to a year 1900 or earlier to clearly distinguish them from unmodified data. Shifting dates is recommended, but not required.
Additional fields can include external behavioural measures relevant to the scan. For example vigilance questionnaire score administered after a resting state scan.

I added **emphasis** above.

If I interpret this correctly, the `aqc_time` column is optional anyways and we don't need to include it, if we don't have a value for it. So I vote for addressing this here:

https://github.com/mne-tools/mne-bids/blob/8e833a97937c51163720bc101abf3e448476cca0/mne_bids/mne_bids.py#L179-L188

... If n/a, drop `acq_time` column from `.tsv`.

The alternative is to open an issue on https://github.com/incf/bids-validator.
Perhaps doing both is worthwhile.

**Comment by jasmainak** [comment]
yes, let's raise an issue on bids-validator. `n/a` is allowed in `tsv` files. This is no different ...
